src/prisma/schema.prisma

```diff
@@ -2055,3 +2057,27 @@ model RecoveryCode {
   code      String
   otp       Otp?
 }
+
+model CharmWallet {
+  id                   String             @id @default(uuid()) @db.Uuid
+  balance              Int                @default(0)
+  userId               String?            @unique @db.Uuid
+  spaceId              String?            @unique @db.Uuid
+  user                 User?              @relation(fields: [userId], references: [id], onDelete: Cascade)
+  space                Space?             @relation(fields: [spaceId], references: [id], onDelete: Cascade)
+  charmTransactionFrom CharmTransaction[] @relation("fromWallet")
```

Comment:
suggestion: just call these 'transactionFrom' or maybe 'transactionsFrom' since its a list. The charm part is implied since this is a charmwallet